__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -84,6 +84,15 @@ export default {
             default: "offWhite",
         },
     },
+    decorators: [
+        (Story) => {
+            React.useEffect(() => {
```

Comment:
Would it be helpful to initialize the announcer for all stories? Or on a per component level? If we want to have it for all stories, we could configure a global decorator.

If we'd like to keep it on a per component level, should we initalize the announcer for the BirthdayPicker too since it uses SingleSelect?

Reply:
Good call. I moved it to a global decorator and added a `targetElement` prop so the Live Regions could be injected in a more precise place in Storybook than `document.body`!

Reply:
One thing to note, the global decorator means the Announcer is injected into every story. There are a lot of Chromatic changes as a result (non-visual as it is screen-reader hidden, but still)!